internal/api/va/list/list.go

```diff
@@ -21,7 +28,22 @@ func (HTTPOperation) SetAuthKey(key string) {
 
 // SetRequestURLArguments sets the appropriate url arguments for the HTTP operation
 func (HTTPOperation) SetRequestURLArguments(args []string) error {
-	id = args[0]
+	return nil
+}
+
+// SetRequestURLArgumentsFromFlags sets the appropriate url arguments from the command flags
+func (HTTPOperation) SetRequestURLArgumentsFromFlags(flags *pflag.FlagSet) error {
```

Comment:
I think we should try to avoid using anything from the cobra package as parameters. This "layer" in the API package only cares about making HTTP operations based on generic inputs.

Reply:
I.e. we should put the each flag as a parameter in SetRequestURLArguments